.github/actions/build-and-push-branch/action.yml

```diff
@@ -4,6 +4,9 @@ inputs:
   branch_version_tag:
     description: 'Used to tag jars and docker images with a branch-specific version (should use the form "dev-<commit_hash>" to pass AirbyteVersion validation)'
     required: false
+  dockerhub_username:
+    description: "Used to log in to dockerhub for pushing images"
+    required: true
```

Comment:
Can you clarify what you mean? Users won't have to type anything in because the workflow will call this action with the stored DOCKER_HUB_USERNAME as the input

Reply:
Ahh gotcha, yeah I don't think we have a current use case for that, good thing to look out for though!